src/consumerauthorization/5_0_0/sd/authorization-token-management/authorization-token-management_sd.tex

```diff
@@ -0,0 +1,427 @@
+Input and output types are only denoted when accepted or returned, respectively, by the operation in question. All abstract data types named in this section are defined in Section 3.
+
+\phantomsection
+\fsubsection{generate-tokens}{AuthorizationTokenGenerationListMgmtRequest}{AuthorizationTokenMgmtListResponse}{}{ErrorResponse}
+
+Operation \textit{generate-tokens} verifies the given consumer systems' permissions to the targeted service/service-operation instance and produces expiring tokens of defined types associated with the consumer system, service provider system and  service instance triplets. The operation returns the generated tokens and the belonged details.
+
+\fsubsection{query-tokens}{AuthorizationTokenQueryRequest}{AuthorizationTokenMgmtListResponse}{}{ErrorResponse}
+
+Operation \textit{query-tokens} lists the access tokens that match the filtering requirements. The query data must meet the following criteria:
+
+\begin{itemize}
+    \item The operation returns results in pages. There are default page data settings, but the requester can provide a custom specification.
+    \item If page number is specified, the page size must be specified as well and vice versa.
+    \item In some Local Clouds there is a maximum page size.
+    \item If a filter expects a list, there is an OR relation between the elements of the filter.
+    \item There is an AND relation between different kind of filters
+\end{itemize}
+
+\fsubsection{revoke-tokens}{AuthorizationTokenRemoveRequest}{OperationStatus}{}{ErrorResponse}
+
+Operation \textit{revoke-tokens} deletes the access token records associated with the given token references. In case of simple tokens this operation can close a session between a consumer and provider.
+
+\fsubsection{add-encryption-keys}{AuthorizationEncryptionKeyMgmtListRequest}{AuthorizationEncryptionKeyMgmtListResponse}{}{ErrorResponse}
+
+Operation \textit{add-encryption-keys} saves and stores \pref{String} key and encryption algorithm identifier pairs belonging to the given provider systems. If the given algorithm is using any addition (besides the encryption key) for the encryption process, such as salt or initialization vector for example, then this addition is returned to the requester system. Any time when a self-contained token is generated that is associated with a provider that has an encryption key saved, the token will be provided to the token requester encrypted, using the specified key and encryption algorithm.
+
+\fsubsection{remove-encryption-keys}{AuthorizationEncryptionKeyRemoveMgmtRequest}{OperationStatus}{}{ErrorResponse}
+
+Operation \textit{remove-encryption-keys} deletes the stored encryption key and algorithm identifier pairs associated with the given provider systems. The result of this operation is that further tokens generated to any service of the affected provider systems, won't be encrypted. 
+
+\clearpage
+
+\section{Information Model}
```

Comment:
The ordering is not following the other documents. This is the good ordering I think: 

1.  AuthorizationTokenGenerationListMgmtRequest
2.  Identity
3.  AuthorizationTokenGenerationMgtmRequest
4.  AccessTokenScope
5.  AuthorizationTokenMgmtListResponse
6.  AuthorizationTokenMgmtResponse
7.  ErrorResponse
8.  AuthorizationTokenQueryRequest
9.  AuthorizationTokenRemoveRequest
10.  AuthorizationEncryptionKeyMgmtListRequest
11.  AuthorizationEncryptionKeyMgmtRequest
12.  AuthorizationEncryptionKeyMgmtListResponse
13.  AuthorizationEncryptionKeyMgmtResponse
14.  AuthorizationEncryptionKeyRemoveMgmtRequest